### ISSUE TITLE
Add TCAS and GPWS

### ISSUE THREAD

**Issue by SetupDotExe** [enjarai/do-a-barrel-roll#32]
Please add a
"WOOP WOOP Terrain Terrain Pull Up Pull Up"
sound when you are about to hit a Mountain in front of you.

Of course this should be turned off by default in the config.

**Comment by enjarai** [comment]
Amazing idea, I'll see what I can do.

**Comment by craftish37** [comment]
_(Reposting, I don't know how the hell I managed to switch issues...)_
Love the idea, here's some matter to work with !
https://www.youtube.com/watch?v=fbfVGIBcD8c
https://en.wikipedia.org/wiki/Terrain_awareness_and_warning_system
https://en.wikipedia.org/wiki/Ground_proximity_warning_system

---

In the same line, some infos provided by tools may be as funny as useful, I was thinking of an [altimeter](https://en.wikipedia.org/wiki/Altimeter), an [attitude indicator](https://en.wikipedia.org/wiki/Attitude_indicator) and maybe even a [variometer](https://en.wikipedia.org/wiki/Variometer), all providing in-game infos in real time (more [here](https://en.wikipedia.org/wiki/Flight_instruments))

Yeah, even for a funny feature, that does sounds fancy...

**Comment by enjarai** [comment]
I'm thinking this should maybe go into a separate addon mod. It'd be quite a big thing if I wanted to do it properly. Maybe something similar to [this mod](https://www.curseforge.com/minecraft/mc-mods/flighthud).

**Comment by craftish37** [comment]
Yeah I do agree, however should make it way easier to read, this mod is occupying way too much screen space

**Comment by carson-coder** [comment]
What if we also added auto pilot

**Comment by craftish37** [comment]
[Here](https://github.com/cabaletta/baritone/pull/4040) you go man 😂
Probably the craziest mod I've ever seen, but I say that every time so...

**Comment by carson-coder** [comment]
They added elytra autopilot to baritone. _If you give a guy a computer, he makes a flight Sim in minecraft better than flying a real plane_

**Comment by carson-coder** [comment]
I am making a mod that adds this. Its sound and one bit of text
